packages/editor/src/editor/Editable.tsx

```diff
@@ -1037,13 +934,45 @@ export const PortableTextEditable = forwardRef<
     [onDragLeave, editorActor, slateEditor],
   )
 
+  const callbackRef = useCallback(
+    (node: HTMLDivElement | null) => {
+      if (typeof forwardedRef === 'function') {
+        forwardedRef(node)
+      } else if (forwardedRef) {
+        forwardedRef.current = node
+      }
```

Comment:
It would be nice if we one day could get rid of the ref forwarding as well (as it's so difficult to optimize and narrow scope if we always have to make everything available to the parent component) 🥲

Reply:
This would be a breaking change, right? But if it's worth it, it's worth it!